CHANGELOG.md

```diff
@@ -61,6 +61,9 @@ Versioning](https://semver.org/spec/v2.0.0.html) for `libnopegl`.
   opened
 - Text outline is now by default on the outer edge, and thus doesn't affect the
   shape of the characters anymore
+- Texture2D.{width,height}, Texture2DArray.{width,height,depth},
+  Texture3d.{width,height,depth}, TextureCube.size, TextureView.layer are now
+  unsigned
```

Comment:
This commit includes changes in the public API header which aren't documented (and probably belong in a separate commit)

Reply:
Thanks.

> [!NOTE]
> There is no need to segment so much, `Texture2D.{width,height}` as a whole is just fine